saved-sites/saved-sites-api/src/main/java/com/duckduckgo/savedsites/api/SavedSitesRepository.kt

```diff
@@ -53,6 +53,8 @@ interface SavedSitesRepository {
      */
     fun getFolderContentSync(folderId: String): Pair<List<Bookmark>, List<BookmarkFolder>>
 
+    fun getAllFolderContentSync(folderId: String): Pair<List<Bookmark>, List<BookmarkFolder>>
```

Comment:
disagree, there’s a specific method for Favourites and another one for Bookmarks. If one day we support folder within Favourites then it might be the case, otherwise not.

Reply:
Want to know more about this one. Seems confusing to me be able to call `getAllFolderContentSync(folderId)`, where folderId is `favorites_root`, and return bookmarks.